crc/__init__.py

```diff
@@ -27,7 +27,6 @@
 app.config.from_object('config.default')
 
 if "TESTING" in os.environ and os.environ["TESTING"] == "true":
-    app.config.from_object('config.testing')
```

Comment:
are you sure you want to remove this?

Reply:
No, I am not sure.
But we are loading the testing configuration twice.